uiowa/uiowa
# Classrooms - Room Schedule Pager

Resolves https://github.com/uiowa/uiowa/issues/6476

_Note: "Mini" pager is a views thing. Not sure it is available for something like this._

# Options to decide:

- how many to show per page
- how many page buttons to show in between first/last buttons


<img width="616" alt="Screenshot 2023-06-12 at 3 03 13 PM" src="https://github.com/uiowa/uiowa/assets/4663676/49fc8b6b-d4d7-4979-8ff4-a0b4c82dff78">

# To Test

```
ddev blt ds --site=classrooms.uiowa.edu && ddev drush @classrooms.local uli ucc-1100
```
- Confirm pager is loading and showing the correct events.
- Try find hole in caching strategy
